src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -82,6 +88,19 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
             $data['tests'] = array_keys($data['tests']);
             $data['loader_paths'] = $this->getLoaderPaths();
+            if ($wrongBundles = $this->findWrongBundleOverrides()) {
+                $messages = $this->buildWarningMessages($wrongBundles);
+                $data['warnings'] = array_reduce(
```

Comment:
I am not sure I understand for what we need `array_reduce()` here. `$messages` already is just an array of strings, isn't it?

Reply:
You're right. It was just dead code. Done